src/PointwiseFunctions/InitialDataUtilities/Background.hpp

```diff
@@ -0,0 +1,55 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <pup.h>
+
+#include "Parallel/CharmPupable.hpp"
+
+namespace elliptic::analytic_data {
+/*!
+ * \brief Subclasses supply variable-independent background data for an elliptic
+ * solve.
+ *
+ * Examples for background fields are a background metric, associated curvature
+ * quantities, matter sources such as a mass-density in the XCTS equations, or
+ * just a source function \f$f(x)\f$ in a Poisson equation \f$\Delta u =
+ * f(x)\f$.
+ *
+ * Subclasses must define the following compile-time interface:
+ *
+ * - They are option-creatable.
+ * - They define a `variables` function that provides the fixed-sources in the
+ *   elliptic equations (see `elliptic::protocols::FirstOrderSystem`). The
+ *   function must have this signature:
+ *
+ *   \snippet Test_InitializeFixedSources.cpp background_vars_fct
+ *
+ *   The function does not need to be `static`. It must support being called
```

Comment:
Instead of adding this caveat about "static" all over the place, just don't mark the examples static.  You might need to silence a compiler warning or something.

Reply:
Pushed a fixup for this